types/mui-datatables/index.d.ts

```diff
@@ -2,14 +2,15 @@
 // Project: https://github.com/gregnb/mui-datatables
 // Definitions by: Jeroen "Favna" Claassens <https://github.com/favna>
 //                 Ankith Konda <https://github.com/ankithkonda>
+//                 Herman "Von" Waters IV <https://github.com/hwatersiv>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.4
```

Comment:
Please revert this. There is no reason why it should require TypeScript 3.4. 3.1 was required due to certain structures that were already being used in the definitions, but these changes include nothing that is only included in TS >= 3.2